sharding-jdbc/sharding-jdbc-core/src/test/java/org/apache/shardingsphere/shardingjdbc/jdbc/core/statement/ShardingPreparedStatementTest.java

```diff
@@ -31,6 +31,7 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertNull;
```

Comment:
This import position should be above the import of aasertThat.

Reply:
Is there any documentation stating import orders ? I didn't find any.

Reply:
No, but this import orders are always according to character order.